Hello, thanks for putting time towards fixing this issue. We can't pull these changes in since tests are failing with the following traceback:

```
psychopy/iohub/util/__init__.py:374: in getDeviceNames
    dconfigs = getDeviceDefaultConfig(device_name)
psychopy/iohub/util/__init__.py:327: in getDeviceDefaultConfig
    device_paths = getDevicePaths(device_name)
psychopy/iohub/util/__init__.py:272: in getDevicePaths
    for ep in entry_points()['psychopy.iohub.devices.eyetracker']:
../../../hostedtoolcache/Python/3.10.13/x64/lib/python3.10/importlib/metadata/__init__.py:427: in __getitem__
    return super().__getitem__(name)
E   KeyError: 'psychopy.iohub.devices.eyetracker'
```

--- a/psychopy/iohub/util/__init__.py
+++ b/psychopy/iohub/util/__init__.py
@@ -16,6 +16,7 @@ import collections.abc
 import pathlib
 import psychopy.logging as logging
 import psychopy.plugins as plugins
+from psychopy.plugins.util import getEntryPoints
 from importlib.metadata import entry_points
 from pathlib import Path
 from psychopy.preferences import prefs
@@ -269,7 +270,7 @@ def getDevicePaths(device_name=""):
 
     if 'eyetracker' in device_name.lower():
         # Find entry points targeting psychopy.iohub.devices.eyetracker
-        for ep in entry_points()['psychopy.iohub.devices.eyetracker']:
+        for ep in getEntryPoints('psychopy.iohub.devices.eyetracker', submodules=False, flatten=True):
             # load the target the entry point points to, it could be a class or a module
             try:
                 ep_target = ep.load()

